### DISCUSSION TITLE
Create bank tag from (inventory, warn items)

### DISCUSSION THREAD

**Discussion by naturedamends** [runelite/runelite#16231] [Q&A] [answered]
Should I add this. I'll write it if peoples think its a good idea. I'm not familiar with the code yet. So perhaps i'm missing something. Or if there's any glaring issues with implementing this proposal would be nice to know, if you have the time and knowledge.

Ie after gearing up for a new task and doing a few runs. It would be nice to 

- click a button to create tag from
- choose a selection of (inventory, worn items, ...., another tag)
- enter new tag name

Feel free to suggest other things but i'll only do it if i've got time

**Comment by Enriath** [comment, marked as answer]
This can already be done by right clicking the deposit buttons in the bottom right of the bank interface.

**Reply by naturedamends** [reply]
sexy - thanks

**Reply by naturedamends** [reply]
Any idea about withdrawing them to a certain invent spot.

**Reply by Nightfirecat** [reply]
That's not something we could make, sorry.

**Reply by naturedamends** [reply]
Whats the issue, with it being close to botting?

**Reply by Nightfirecat** [reply]
It would require macroing to be done
